TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_cli.py

```diff
@@ -432,6 +434,36 @@ def test_cli_snapshot_delete_without_acknowledgement_doesnt_run(runner, mocker):
     mock.assert_not_called()
 
 
+def test_cli_snapshot_unregister_repo_with_acknowledgement(runner, mocker):
+    mock = mocker.patch.object(Cluster, 'call_api', autospec=True)
+    mock.return_value.text = "Successfully deleted"
+
+    # Test snapshot status
+    result = runner.invoke(cli, ['--config-file',
+                                 str(VALID_SERVICES_YAML),
+                                 'snapshot',
+                                 'unregister-repo',
+                                 '--acknowledge-risk'],
+                           catch_exceptions=True)
+    assert result.exit_code == 0
+
+    # Ensure the mocks were called
+    mock.assert_called_once()
+
+
+def test_cli_snapshot_unregister_repo_without_acknowledgement_doesnt_run(runner, mocker):
+    mock = mocker.patch.object(Cluster, 'call_api', autospec=True)
+    mock.return_value.text = "Successfully deleted"
+
+    # Test snapshot status
+    result = runner.invoke(cli, ['--config-file', str(VALID_SERVICES_YAML), 'snapshot', 'unregister-repo'], input="n",
+                           catch_exceptions=True)
+    assert result.exit_code == 0
+
+    # Ensure the mocks were called
```

Comment:
nit: "Ensure the mocks were **not** called"

Reply:
Thanks updated